sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -236,10 +253,13 @@ def from_dictionary(cls, options):
       A PipelineOptions object representing the given arguments.
     """
     flags = []
+    params = {}
     for k, v in options.items():
       if isinstance(v, bool):
         if v:
           flags.append('--%s' % k)
+        else:
+          params[k] = v
```

Comment:
Likewise, here I think we should only allow 'True' values, and print a warning to the user when this is not the case.